src/components/TeamMemberTasks/components/TaskCompletedModal.jsx

```diff
@@ -35,41 +39,46 @@ const TaskCompletedModal = React.memo(props => {
     });
 
     const updatedTask = { ...task, resources: newResources };
-    props.updateTask(task._id, updatedTask);
+    props.updateTask(task._id, updatedTask, setIsLoadingTask);
```

Comment:
```suggestion
    props.updateTask(task._id, updatedTask);
```
'setIsLoadingTask' doesn't appear to be needed in this function.

Reply:
Solved